n3c_owl_ingest/n3c_owl_ingest.py

```diff
@@ -245,24 +287,40 @@ def run_ingest(split_by_vocab: bool = False, skip_if_in_release: bool = True, me
 
     # Read inputs
     # - concept_relationship table
-    concept_subsumes_df = pd.read_csv(CONCEPT_RELATIONSHIP_SUBSUMES_CSV, dtype=CONCEPT_RELATIONSHIP_DTYPES).fillna('')
-    concept_subsumes_df = concept_subsumes_df[concept_subsumes_df.invalid_reason == '']
-    concept_subclass_ofs: Dict[List[int]] = {}
-    for row in concept_subsumes_df.itertuples(index=False):
-        # noinspection PyUnresolvedReferences It_doesnt_know_that_row_is_a_namedtuple
-        concept_subclass_ofs.setdefault(row.concept_id_2, []).append(row.concept_id_1)
-    # how_many_parents_gt_1 = [len(x) > 1 for x in concept_subclass_ofs.values()].count(True)  # analysis: 215,489 / ~7m
-    num_subclass_cols = max([len(x) for x in concept_subclass_ofs.values()])  # determined by greatest num parents
+    concept_rel_df = pd.read_csv(concept_relationship_csv_path, dtype=CONCEPT_RELATIONSHIP_DTYPES).fillna('')
+    concept_rel_df = concept_rel_df[concept_rel_df.invalid_reason == '']
+    # todo: include automatic addition of these relationships?
```

Comment:
I have some special logic that can include any OMOP relationship type that has the word 'rx' or 'atc' in it, if we want to activate it. Commented out for now.